js/getMovieList.js

```diff
@@ -1,9 +1,58 @@
 import MOVIES_API_KEY from "./key.js";
-import NO_POSTER_URL from "./constants.js"
+import {NO_POSTER_URL} from "./constants.js";
+import {headerRender} from "./headerRender.js";
 
-const movieListById = document.querySelector('#movie-list'),
-     searchBarInput = document.querySelector('.search-bar__input'),
-     searchBtn = document.querySelector('.search-bar__button');
+
+const body = document.querySelector('body');
+//At first function that creates header initializes
+headerRender();
+
+
+/**
+ * This function renders a search bar proceduraly
+ */
+function searchBarRender() {
+    const SBSection = document.createElement('section'),
```

Comment:
do not use abbreviations in your code for naming variables unless it is widely accepted and easy to understand, e.g. "url"
https://medium.com/technogise/dont-use-abbreviations-while-naming-variables-49c041e05e96